src/tests/unit/AccessTokenHandler.test.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import { mock } from "jest-mock-extended";
+import { AccessTokenRequestProcessor } from "../../services/AccessTokenRequestProcessor";
+import { lambdaHandler } from "../../AccessTokenHandler";
+import { CONTEXT } from "./data/context";
+import { HttpCodesEnum } from "../../models/enums/HttpCodesEnum";
+import { VALID_ACCESSTOKEN } from "./data/accessToken-events";
+
+const mockedAccessTokenRequestProcessor = mock<AccessTokenRequestProcessor>();
+
+jest.mock("../../services/AccessTokenRequestProcessor", () => {
+	return {
+		AccessTokenRequestProcessor: jest.fn(() => mockedAccessTokenRequestProcessor),
+	};
+});
```

Comment:
I actually don't think we need this if we're doing `AccessTokenRequestProcessor.getInstance = jest.fn().mockReturnValue(mockedAccessTokenRequestProcessor);` in every test